WcaOnRails/app/webpacker/components/EditEvents/store/reducer.js

```diff
@@ -127,12 +135,14 @@ const reducers = {
       (round) => (round.timeLimit?.cumulativeRoundIds ? {
         timeLimit: {
           ...round.timeLimit,
-          cumulativeRoundIds: round.timeLimit.cumulativeRoundIds.filter(
-            (roundId) => ![payload.roundId, ...payload.timeLimit.cumulativeRoundIds].includes(roundId)
-          ),
+          cumulativeRoundIds: round.timeLimit.cumulativeRoundIds.filter((roundId) => ![
+            payload.roundId,
+            ...payload.timeLimit.cumulativeRoundIds,
+          ].includes(roundId)),
```

Comment:
ESLint changed the line layout here, because the previous "everything on one line" version was too long.
De facto it's just indentation changes, no actual code has changed.